x/blob/ante/blob_share_decorator.go

```diff
@@ -75,7 +76,8 @@ func (d BlobShareDecorator) getMaxSquareSize(ctx sdk.Context) int {
 
 // getSharesNeeded returns the total number of shares needed to represent all of
 // the blobs described by blobSizes.
-func getSharesNeeded(blobSizes []uint32) (sum int) {
+func getSharesNeeded(base uint32, blobSizes []uint32) (sum int) {
+	sum = share.CompactSharesNeeded(base)
```

Comment:
[optional] rename base to something more meaningful

```suggestion
func getSharesNeeded(pfbTxSize uint32, blobSizes []uint32) (sum int) {
	sum = share.CompactSharesNeeded(pfbTxSize)
```